_posts/2021-06-25-app.md

```diff
@@ -0,0 +1,103 @@
+
+### Datenerhebung
+
+Zur späteren Analyse zeichnet die App die Routen der Tester auf und wird diese später anonymisiert an uns übertragen. Dabei ist uns besonders wichtig, dass aus diesem Datensatz der tatsächliche Wohnort / Arbeitsstädte unserer Tester:Innen nicht hervorgeht. Hierzu wird vor Beginn der Datenübertagung hier dazu ein Artikel erscheinen. Aus diesem Datensatz lässt sich die tatsächliche Reisezeit und umfahrene Hindernisse entnehmen.
+
+Zum Einstieg in die App fragen wir auch die Fahrzeugdimensionen und Art des Fahrzeuges ab. Diese Daten lassen wir auch in das Routing einfließen, sodass beispielsweise Poller mit zu niedriger Breite umfahren werden können.
+
+## Beta-test
+
+<img alt="Werbe Banner CargoRocket App" src="/assets/images/app-Banner-1.png" style="max-width: 100%;" />
+
+Um ein sinnvolles Lastenradrouting zur verfügung stellen zu können benötigen wir dein Feedback. Nach vielen Nachforschungen und Tests können wir nur mit echten Streckenverläufen und echten Fahrer:Innen das Routing weiter verfeinern.
+
+Um mitzumachen registriere dich unter: https://play.google.com/apps/testing/de.cargorocket
+
+### Feedback
+
+Uns ist bewusst, dass der aktuelle Stand der App bei weitem noch nicht perfekt ist. Hier gibt es einiges zu verbessern und dazu bist du gefragt.
+Damit wir dein Feedback so schnell wie möglich in die App und das Routing integrieren können wären folgende Punkte wichtig:
+
+* Das Routing ist momentan optimiert für kommerzielle Lastenradlogistik und unterscheidet sich daher teilweise stark von einem profil für Freizeit-/Privatbedarfs-Radverkehr.
+
+* In der App befindet sich unter Einstellungen > Feedback eine Feedbackfunktion, die für kurze Hinweise am besten genutzt wird. Auch während des Routings kann mit der "Stern"-Funktion feedback zum aktuellen Streckenabschnitt gegeben werden. Für komplexeres Feedback sendet uns gerne eine [Mail](mailto:team+playweb@cargorocket.de?subject=%5BAPP-BETA%5D%20Feedback).
+
+* Gibt es Schlatflächen und Flows in der App die dir unintuitiv oder übermäßig komplex vorkommen?
+
+* Sind die vorgeschlagenen Routen aus deiner Sicht sinnvoll und falls nein, welche Kriterien sind hierbei entscheidend?
+
+* Würdest du eine Routing App für deine Fahrten generell nutzen? Welche Faktoren sind hier für dich wichtig?
+
+## Technische Details
+
+Im folgenden Abschnitt findet sich eine kurze Beschreibung des technischen Aufbaus unserer App. Diese Beschreibung ist an Entwickler gerichtet und daher teilweise sehr technisch.
+
+Um möglichst schnell beide großen mobilen Plattformen (IOS, Android) bedienen zu können setzten wir bei der Technologiewahl auf ein cross-platform Framework. Zur Auswahl standen dabei unter Anderen [NativeScript](https://nativescript.org/), [Cordova](https://cordova.apache.org/), [Flutter](https://flutter.dev/) und [ReactNative](https://reactnative.dev/).
+Ausschlaggebendes Kriterium bei der Entscheidung war für uns die Unterstützung von Mapbox Navigation.
```

Comment:
Maybe you could write here that you decided on going with React Native?

Reply:
and why?